config/locales/en.yml

```diff
@@ -233,6 +233,20 @@ en:
               <<: *participant
             reason:
               <<: *participant_reason
+        participants/change_schedule:
+          attributes:
+            course_identifier:
+              <<: *participant_course_identifier
+            participant:
+              <<: *participant
+              already_withdrawn: Cannot perform actions on a withdrawn participant
```

Comment:
Can we just use the one from the participant errors above? maybe one for Mark but I would expect the errors to be the same